pandas/core/indexes/multi.py

```diff
@@ -882,15 +882,32 @@ def _get_level_values(self, level):
     def get_level_values(self, level):
         """
         Return vector of label values for requested level,
-        equal to the length of the index
+        equal to the length of the index.
 
         Parameters
         ----------
-        level : int or level name
+        level : int or str
+            ``level`` is either the integer position of the level in the
+            MultiIndex, or the name of the level.
 
         Returns
         -------
         values : Index
+
+        Examples
```

Comment:
Note that I've made the examples a bit simpler. No need to demonstrate CategoricalIndex etc. here...